packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -538,6 +543,23 @@ describe('Table', function () {
       </Table>
     );
 
+    let renderMany = () => render(
+      <Table selectionMode="none">
+        <TableHeader columns={columns} columnKey="key">
+          {column =>
+            <Column>{column.name}</Column>
+          }
+        </TableHeader>
+        <TableBody items={manyItems} itemKey="foo">
+          {item =>
+            (<Row>
+              {key => <Cell>{item[key]}</Cell>}
+            </Row>)
+          }
+        </TableBody>
+      </Table>
+    );
+
     let focusCell = (tree, text) => tree.getByText(text).focus();
     let moveFocus = (key, opts = {}) => fireEvent.keyDown(document.activeElement, {key, ...opts});
```

Comment:
this should be wrapped in an `act`

Reply:
hmm I don't get any warnings about these...

Reply:
weird, i assume they induce state changes, but if you aren't seeing any warnings from them... leave it i guess
`act` still confuses me